docs/git/index.md

```diff
@@ -0,0 +1,220 @@
+---
+id: index
+title: Introduction
+sidebar_label: Introduction
+---
+
+Most software engineers use a tool called Git to organise their code, and collaborate with other people.
+
+Git can be used for many things, and we will see lots of them throughout the course.
```

Comment:
I think it might be better to replace this sentence for something like:

"Git is a version control tool useful to keep the history of your code changes."